backend/routes/auth.py

```
@@ -0,0 +1,276 @@
+
+        # Clear cookie
+        response.delete_cookie(key="admin_session")
+
+        return {"success": True, "message": "Logged out successfully"}
+
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Logout failed: {str(e)}")
+
+
+@router.post("/change-password")
+async def change_password(request: ChangePasswordRequest, user: Dict = Depends(get_current_user)) -> Dict:
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)